isthmus/src/main/java/io/substrait/isthmus/SubstraitRelNodeConverter.java

```diff
@@ -1,5 +1,6 @@
 package io.substrait.isthmus;
 
+import static io.substrait.isthmus.SqlConverterBase.EXTENSION_COLLECTION;
 import static io.substrait.isthmus.SqlToSubstrait.EXTENSION_COLLECTION;
```

Comment:
I am surprised there wasn't a compile error with two static imports of the same name. I guess because they actually refer to the same constant. I think there should only be one import here though.